app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="log-in-container">
+
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), data: { turbo: false }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+
+  <div class="field">
+      <%= f.label :name %><br />
+      <%= f.text_field :name,autofocus: true, class: "form-control my-3" %>
+    </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control my-3" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", class: "form-control my-3" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "form-control my-3" %>
```

Comment:
- [ ] From the UI, I can't able to signup, It's giving me this error which i have attached below. Please fix this. I have checked your application by making the user through the command line.

![Screenshot from 2022-02-04 00-14-59](https://user-images.githubusercontent.com/22434660/152412147-200586aa-912d-4f73-ae7c-aec8b5d1b3dc.png)

Reply:
- [ ] The error has been fixed. However, I was still not able to log in after registering as a user. I keep getting the below message on the login screen
![Screenshot from 2022-02-04 01-33-49](https://user-images.githubusercontent.com/20245164/152441482-bf4d091c-dcf8-423a-9d44-c0d97518629b.png)
Kindly fix it and ensure that a user is redirected after a successful login.

Reply:
Hello @oloomoses, we are using email confirmation that's why you weren't able to log In, however, I changed the root route now and the send notification for email confirmation can be seen.